### PR TITLE
Support Nextcloud 27

### PR DESCRIPTION
**Pull request by provokateurin**
Closes https://github.com/provokateurin/nextcloud-neon/issues/391

The news app sadly hasn't been updated for/is broken on 27. It only affects the web UI which we don't care about (See https://github.com/nextcloud/news/issues/2242). @Leptopoda should we merge regardless or wait for the new app to get fixed? The linked issue sounded a bit like resolving the problem could take some time...

### CODE REVIEW

File: packages/nextcloud/test/webdav.dart
Comment (Leptopoda) [inline packages/nextcloud/test/webdav.dart:1]: Does the default NC now include more files or why is this change required

**Review by Leptopoda — APPROVED** [review]
LGTM;

I've been using neon with 27 for a few days now and it works :)

**Comment by Leptopoda** [comment]
If the news app works for us we should go forward with the update.

It's at the users discretion to update to 27 with incomplete news web support (I for example don't use news anymore and thus updated to 27 fairly quickly).

File: packages/nextcloud/test/webdav.dart
Comment (provokateurin) [reply packages/nextcloud/test/webdav.dart:1]: Yep
